packages/contentstack-auth/README.md

```diff
@@ -5,44 +5,40 @@ It is Contentstack’s CLI plugin to perform authentication-related activities.
 [![License](https://img.shields.io/npm/l/@contentstack/cli)](https://github.com/contentstack/cli/blob/main/LICENSE)
 
 <!-- toc -->
-
-- [@contentstack/cli-auth](#contentstackcli-auth)
-- [Usage](#usage)
-- [Commands](#commands)
+* [@contentstack/cli-auth](#contentstackcli-auth)
+* [Usage](#usage)
+* [Commands](#commands)
 <!-- tocstop -->
 
 # Usage
 
 <!-- usage -->
-
 ```sh-session
 $ npm install -g @contentstack/cli-auth
 $ csdx COMMAND
 running command...
 $ csdx (--version)
-@contentstack/cli-auth/1.3.13 darwin-x64 node-v20.7.0
+@contentstack/cli-auth/1.3.13 darwin-arm64 node-v18.11.0
```

Comment:
@shafeeqd959 It's better to keep the updated version if there are no major changes in readme files.